### ISSUE TITLE
Allow to create backends with a single core

### ISSUE THREAD

**Issue by mihaiconstantin** [mihaiconstantin/parabar#71]
This is regarding Jonas' request in https://github.com/mihaiconstantin/parabar/issues/53#issuecomment-2467538902.

**Comment by mihaiconstantin** [comment]
To maintain backwards compatibility, I propose we implement this as a package option, say `cores_minimum`. Currently, `parabar` shows a warning if less than two cores are used, so it would be sensible to set `cores_minimum = 2` as default.

Then, users can customize this to their needs as follows:

```r
# Adjust the minimum number of backend cores.
set_option("cores_minimum", 1)

# Start a backend.
backend <- start_backend(cores = 1)

# At this point this will not show a warning anymore.

# Dispose the backend.
stop_backed(backend)
```

Similarly:

```r
# Adjust the minimum number of backend cores.
set_option("cores_minimum", 4)

# Start a backend.
backend <- start_backend(cores = 2)

# This will show a warning and, should they are available, set to `4` cores.

# Dispose the backend.
stop_backed(backend)
```

Would this address the issue, @jmbh?

**Comment by jmbh** [comment]
Hi Mihia, 

Apologies for the slow response!

Thanks, yes, that sounds like a good solution and it would be great if you could include it in the package!

Best wishes,
Jonas

**Comment by mihaiconstantin** [comment]
@jmbh, this is now fixed in #78. Please note that there is no need for `set_option("cores_minimum", 1)` as previously discussed. You can just go ahead and use `start_backend(cores = 1)` as usual.

This will be included in the next `parabar` release `v1.3.0`, which I will submit to `CRAN` this week. Hope this helps!

**Comment by jmbh** [comment]
Excellent, thanks a lot!

Jonas
